portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/questions/QuestionsAllQuestionsList.testcase

```diff
@@ -0,0 +1,129 @@
+@component-name = "portal-questions"
+definition {
+
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Questions";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		JSONLayout.addPublicLayout(
+			groupName = "Guest",
+			layoutName = "Questions Page");
+
+		JSONLayout.addWidgetToPublicLayout(
+			column = 2,
+			groupName = "Guest",
+			layoutName = "Questions Page",
+			widgetName = "Questions");
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if (${testPortalInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			ProductMenu.gotoPortlet(
+				category = "Content & Data",
+				portlet = "Message Boards");
+
+			if (IsElementPresent(locator1 = "Questions#QUESTIONS_LIST")) {
+				Questions.tearDown();
+			}
+
+			PagesAdmin.tearDownCP();
+		}
+	}
+
+	@description = "Story ID: LPS-153187 | Test Summary: Verify if the user can access all questions by dropdown."
+	@priority = 5
+	test CanAccessAllQuestionsOnDropdown {
+		task ("Given that add two categories and questions") {
+			JSONMBMessage.addCategory(
+				categoryDescription = "Description",
+				categoryName = "Questions Test Topic",
+				userEmailAddress = "test@liferay.com");
+
+			JSONMBMessage.addCategory(
+				categoryDescription = "Description",
+				categoryName = "Questions Category",
+				userEmailAddress = "test@liferay.com");
+
+			JSONQuestions.addQuestion(
+				bodyField = "Content AddQuestion",
+				categoryName = "Questions Test Topic",
+				groupName = "Guest",
+				titleField = "Question1");
+
+			JSONQuestions.addQuestion(
+				bodyField = "AddQuestion2",
+				categoryName = "Questions Category",
+				groupName = "Guest",
+				titleField = "Question2");
+		}
+
+		task ("And go to Questions Test Topic") {
+			Navigator.gotoPage(pageName = "Questions Page");
+
+			LexiconCard.clickCard(card = "Questions Test Topic");
+		}
+
+		task ("When select all questions on dropdown") {
+			Click(
+				key_topicName = "Questions Test Topic",
+				locator1 = "Questions#QUESTIONS_BREADCRUMB_PARENT_TOPIC_BUTTON");
+
+			Click(
+				key_topicName = "All Questions",
+				locator1 = "Questions#QUESTIONS_BREADCRUMB_TOPIC_DROPDOWN_OPTION");
+		}
+
+		task ("Then can see all questions") {
+			AssertElementPresent(
+				key_questionTitle = "Question1",
+				locator1 = "Questions#QUESTIONS_TITLE");
+
+			AssertElementPresent(
+				key_questionTitle = "Question2",
+				locator1 = "Questions#QUESTIONS_TITLE");
+		}
```

Comment:
Please call the assert only once and assert both items.
e.g. 
```
for (var questionsList : list "Question1,Question2") {
AssertElementPresent(
	key_questionTitle = ${questionsList},
	locator1 = "Questions#QUESTIONS_TITLE");
}
```

Reply:
Done